### ISSUE TITLE
Imagesize (Max. Width) restrictions

### ISSUE THREAD

**Issue by micheledallatorre** [johanneszab/TumblTwo#7]
Hello,
would it be possible to remove, or at least increase, the _Imagesize (Max. Width)_ to more than 1280px?

![image](https://cloud.githubusercontent.com/assets/4906504/20517539/c4eb7dd4-b09b-11e6-8254-32cfeb68caad.png)

Thanks a lot

**Comment by Scaarus** [comment]
This is the largest image size Tumblr supports, isn't it?

**Comment by johanneszab** [comment]
Yes, it it.
You can apparently display larger images, but then they are hosted externally on sites like imgur.com.

It's probably worth adding support for youtube for TumblThree. There already is a library from someone in C#. We basically just have to check if it's a youtube url and then downloaded the video from there with that library. 

Unlike for pictures, youtube hosts almost all videos nowadays.

**Comment by micheledallatorre** [comment]
@Scaarus you're right, see documentation here https://www.tumblr.com/docs/en/api/v2#photo-posts

@johanneszab exactly, I am pretty sure I saw some Tumblr posts where I could click on the photo post and by opening it in a new browser tab, the picture appearead to be bigger than 1280px...
